docs/user-roles.rst

```diff
@@ -44,3 +53,58 @@ also perform every available operation inside that project.
     organization.
 
 A *project member* has read-only access to the project.
+
+
+.. _user-ops:
+
+User operations
+===============
+
+
+.. _system-user:
+
+``System`` user
+---------------
+
+In CrateDB Cloud, the backend uses a user called ``system`` in order to perform
+Cloud cluster upgrades, backups and scaling functions, among other things.
+
+.. WARNING::
+    The user ``system`` is essential for CrateDB Cloud to function as intended.
+    While it is not normally accessible through the CrateDB Console or the
+    Croud CLI, it can be accessed through the CrateDB Admin UI or any other SQL
+    client. It is important not to edit or delete this user in any way.
+    Otherwise, the functioning of Cloud clusters may be compromised.
+
+
+.. _crate-user:
+
+``Crate`` user
+--------------
+
+In CrateDB, when you create a cluster node (through whatever method), a
+``crate`` user is automatically generated. ``crate`` is a superuser and can
+perform all possible operations. For CrateDB Cloud purposes, however, this user
+is not exposed to the Cloud customer, who instead creates their own users
+according to the user roles and privileges described above. (For information on
+how to do so, see our documentation on `adding users`_).
```

Comment:
I think you mix up two different user types here: On the one hand there are Cloud users, and on the other hand there are CrateDB users.

The former are the ones you log in with to the console and has privileges such as org member/admin, project member/admin.

The latter are `crate`, `system` and the one that is specified upon cluster creation. Both the `crate` and `system` user are visible to the user generated user, but cannot be used. The user generated user has [AL privileges](https://crate.io/docs/crate/reference/en/latest/admin/privileges.html#al).

https://crate.io/docs/crate/reference/en/latest/admin/user-management.html
> When CrateDB is started, the cluster contains one predefined superuser. This user is called crate. It is not possible to create any other superusers.